src/Components/Header/index.jsx

```diff
@@ -6,22 +6,25 @@ const Header = () => {
     const [displayMenu, setDisplayMenu] = useState(false)
 
     return (
-        <div className="h-100 w-100">
-            <div className="row d-flex justify-content-between align-items-center border border-secondary bg-secondary text-white">
-                <button onClick={() => {setDisplayMenu(!displayMenu)}} className="col-2 mx-3 border border-black btn btn-secondary d-md-none text-center" type="button" data-bs-toggle="dropdown" aria-expanded="false">MENU</button>
+        <>
+            <div className="col-12 d-flex justify-content-between align-items-center border border-secondary bg-secondary text-white">
+                <div className="col-2 d-md-none">
```

Comment:
Not sure if this is just GitHub but this indentation is odd. Move everything in the div to same indent.